### PR TITLE
add c# enums doc

### PR DESCRIPTION
**Pull request by jrood**
### Description

Doc added to explain enums in C# under `content/c-sharp/concepts/enums/enums.md`

I included examples of an enum both without specific number assignments, along with a codebyte example of using an enum for different log types.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/enums/enums.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  - 'Data Types'
+  - 'Arrays'
+  - 'Vectors'
+  - 'Pointers'
+  - 'Memory'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/enums/enums.md:18]: We need an intro paragraph that summarizes what an enum is. It should at the very least be a copy of the description above. (With the first occurrence of the term "enum" in bold.)

File: content/c-sharp/concepts/enums/enums.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  Leonardo, // 0
+  Michelangelo, // 1
+  Donatello, // 2
+  Raphael // 3
+}
+
+Console.WriteLine((int)NinjaTurtle.Raphael == 3); // prints "True"
+```
+
+However, you can also choose to assign specific number values.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/enums/enums.md:34]:
```suggestion
However, specific number values can be assigned.
```
We avoid first and second person pronouns in Docs.

File: content/c-sharp/concepts/enums/enums.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+```cs
+enum BreadCount {
+  Dozen = 12,
+  BakersDozen = 13,
+  Gross = 144,
+  GreatGross = 1728
+}
+```
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/enums/enums.md:44]: I'd like to see an additional example. (Not a codebyte, just an additional example beforehand.)

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jrood I did a first review and left a few comments. Mostly about additional information to be added to this entry.

**Comment by jrood** [comment]
Thanks for the feedback @SSwiniarski ! I believe I've resolved all the issues that you mentioned. Let me know if there's anything else!

File: content/c-sharp/concepts/enums/enums.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+The following example uses an enum to identify whether a given place is a city, country, or planet.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/enums/enums.md:47]: Add an `## Example` header before this line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jrood One more thing.

File: content/c-sharp/concepts/enums/enums.md (hunk 4 above)
Comment (jrood) [reply content/c-sharp/concepts/enums/enums.md:49]: ah thanks, done!

**Review by SSwiniarski — APPROVED** [review]
@jrood Looks good for a second review.

File: content/c-sharp/concepts/enums/enums.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+public class Example {
+  public static void Main() {
+    List<Place> places = new List<Place>();
+    places.Add(new Place("Tokyo", PlaceType.City));
+    places.Add(new Place("Canada", PlaceType.Country));
+    places.Add(new Place("Jupiter", PlaceType.Planet));
+  }
+}
+
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/enums/enums.md:78]:
```suggestion
```

File: content/c-sharp/concepts/enums/enums.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+```cs
+enum NinjaTurtle {
+  Leonardo, // 0
+  Michelangelo, // 1
+  Donatello, // 2
+  Raphael // 3
+}
+
+Console.WriteLine((int)NinjaTurtle.Raphael == 3); // prints "True"
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/enums/enums.md:33]:
```suggestion
Console.WriteLine((int)NinjaTurtle.Raphael == 3); // Prints "True"
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@jrood Just added a few minor edits.

**Comment by jrood** [comment]
Thanks @caupolicandiaz ! I just made the changes you requested.

**Review by caupolicandiaz — APPROVED** [review]
@jrood LGTM

**Comment by SSwiniarski** [comment]
@jrood congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/c-sharp/enums

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Note**: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**